scania-digital-design-system/sdds
# fix(form elements): Opened shadow DOM to be able to read the proper values

**Describe pull-request**  
Shadow DOM has been opened for textfield, textarea and slider.
This means that those elements can now be read as they were "proper" form elements.
A new story

**Solving issue**  
Fixes: #1896

**How to test**  
A new entry "FormTest" under "Utilities" was created, based on the original test-code from Mena.
Open the console and see what happens when the submit-buttons are pressed.
